spec/javascripts/components/stories/history_spec.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import History, {
+  Changes,
+  Header
+} from '../../../../app/assets/javascripts/components/stories/History';
+import HistoryFactory from '../../support/factories/historyFactory';
+
+describe('<History />', () => {
+  const wrapper = mount(<History history={HistoryFactory()} />);
+
+  it('Should render a History', () => {
+    expect(wrapper.find('.history-activity')).toExist();
+    expect('history' in wrapper.props()).toEqual(true);
+  });
+
+  it('Should render a list of Changes', () => {
+    expect(wrapper.find(Changes)).toExist();
+    expect(
+      Object.keys(
+        wrapper
+          .find(Changes)
+          .at(1)
+          .props().changes
+      ).length
+    ).toBe(6);
+  });
+
+  it('Should render a Header', () => {
+    expect(wrapper.find(Header)).toExist();
+    expect(wrapper.find(Header).length).toBe(2);
+    expect(
+      wrapper
+        .find(Header)
+        .at(0)
+        .props().title
```

Comment:
You can make it a little more readable like this:

```suggestion
        .prop('title')
```

The same applies to the `date` on line 42 :smile:

Reply:
same on line 26  :smile:, thanks